README.md

```diff
@@ -20,8 +20,19 @@ This project processes clinical data from https://clinicaltrials.gov/, specifica
 ## Set up
 
 ### For Windows Beginners
-
-*To be written*
+1)Open Windows PowerShell
+2)Type these commands includeing any spaces but not includeing brackets
+```
+cd [PathToRepository] //this takes power shell to where the files you want to run are stored
```

Comment:
1. This is not the correct syntax for comment in powershell.
2. [PathToRepository] should be explained somewhere.